app/_locales/en/messages.json

```diff
@@ -4226,6 +4244,9 @@
   "stake": {
     "message": "Stake"
   },
+  "stakeDescription": {
```

Comment:
is the text same as `buyDescription`, do we need both?

Reply:
This will be different than buyDescription, I have created this key here. Because we need to change it when the copy will be updated from design team. But I don't have the updated copy yet. So, I am kept it same